### PR TITLE
added tldraw on demo app

### PR DESCRIPTION
**Pull request by japsasuncion**

### CODE REVIEW

File: Demo/Client/src/hooks/useTldrawStore.ts [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+            yStore,
+            meta,
+            room: yjsConnector,
+        };
+    }, [opts, yjsConnector, yjsDocument]);
+
+    useEffect(() => {
+        setStoreWithStatus({ status: 'loading' });
+
+        const unsubs: (() => void)[] = [];
```
Comment (SebastianStehle) [inline Demo/Client/src/hooks/useTldrawStore.ts:61]: Can you just rename this variable, I am not big fan of abbreviations

File: Demo/Client/src/hooks/useTldrawStore.ts [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+
+        function handleSync() {
+            // 1.
+            // Connect store to yjs store and vis versa, for both the document and awareness
+
+            /* -------------------- Document -------------------- */
+
+            // Sync store changes to the yjs doc
+            unsubs.push(
```
Comment (SebastianStehle) [inline Demo/Client/src/hooks/useTldrawStore.ts:70]: Is it possible to refactor this into smaller methods. It is pretty long.

**Review by SebastianStehle — COMMENTED** [review]
Thanks a lot. Just two small changes. You can also create the PR into the new-write branch directly.

File: Demo/Client/src/hooks/useTldrawStore.ts (hunk 2 above)
Comment (japsasuncion) [reply Demo/Client/src/hooks/useTldrawStore.ts:70]: I can try. I just copied it from tldraw's [example](https://github.com/tldraw/tldraw-yjs-example/blob/main/src/useYjsStore.ts),

**Pull request closed without merging**